File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {

      HashSet<Integer> map = new HashSet<>();

      for(int i =0; i<nums.length; i++){
        int num = nums[i];
        if(map.contains(num)){
            return true;
        }
        map.add(num);
      }
      return false;
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        freqMap ={}
        for num in  nums:
            freqMap[num]=freqMap.get(num, 0)+1
        bucket = [[] for i in range(len(nums)+1)]
        for num,cnt in freqMap.items():
            bucket[cnt].append(num)
        res =[]
        for i in range(len(bucket)-1, 0, -1):
            for num in bucket[i]:
                res.append(num)
                if len(res)==k:
                    return res



        